config.toml

```
@@ -1,6 +1,8 @@
 [global]
 strategy = 'naive'
-log_level = 'error'
+log_level = 'info'
+telemetry_enabled = true
+telemetry_port = 3001
```

Can we move this into a `telemetry` section?

```suggestion
[telemetry]
enabled = true
port = 3001
```

---

I think makes sense to have it's own section